python/ngraph/__init__.in.py

```diff
@@ -71,3 +73,20 @@ def disable():
 
 def is_enabled():
   return ngraph.ngraph_is_enabled()
+
+
+def backends_len():
+  return ngraph.ngraph_backends_len()
+
+
+def list_backends():
+  len_backends = backends_len()
+  result = (ctypes.c_string * len_backends)(*(None * len_backends))
+  if not ngraph.ngraph_build_backends(result, len_backends):
+    raise Exception("Backends fluctuated while listing")
```

Comment:
Do we expect this to happen?

Reply:
it seems very, very unlikely, but who knows